Src/CSharpier.VSCode/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.2.4]
+- Error output loses characters
```

Comment:
In hindsight, this isn't a release note: it doesn't explain what's changed.
Maybe something like:
```
Remove error output display. It used to lose characters.
```